CRM/Admin/Form/Job.php

```diff
@@ -156,6 +159,13 @@ public function setDefaultValues() {
 
     CRM_Core_DAO::storeValues($dao, $defaults);
 
+    // CRM-17686
+    if (!empty($dao->scheduled_run_date)) {
+      $ts = strtotime($dao->scheduled_run_date);
+      $defaults['scheduled_run_date'] = date('m/d/Y', $ts);
+      $defaults['scheduled_run_date_time'] = date('h:iA', $ts);
```

Comment:
And this can be simplified to 

    $defaults['scheduled_run_date'] = date("Y-m-d H:i:s", $ts);